govc/vcsa/shutdown/reboot.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright (c) 2022 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package shutdown
+
+import (
+	"context"
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/vapi/appliance/shutdown"
+)
+
+type reboot struct {
+	*flags.ClientFlag
+
+	reason string
+	delay  int
+}
+
+func init() {
+	cli.Register("vcsa.shutdown.reboot", &reboot{})
+}
+
+func (cmd *reboot) Register(ctx context.Context, f *flag.FlagSet) {
+	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
+	cmd.ClientFlag.Register(ctx, f)
+
+	f.IntVar(&cmd.delay,
```

Comment:
nit: formatting difference (in the other files you did one line)

Reply:
To adhere to 80col limitper line, updated this to multiline in both the files, reboot.go and poweroff.go

```
f.IntVar(&cmd.delay,
		"delay",
		0,
		"Minutes after which Power off should be initiated.")
```